pkg/plugin/response.go

```diff
@@ -41,10 +42,31 @@ func parseStreamResponse(reader io.Reader) backend.DataResponse {
 	lineField := data.NewFieldFromFieldType(data.FieldTypeString, 0)
 	lineField.Name = gLineField
 
-	scanner := bufio.NewScanner(reader)
+	sc := bufio.NewReader(reader)
```

Comment:
I think about this because I checked the logic of the parsing log lines. I decided to use default values because under the hood the bufio.NewReader has an expandable buffer like the one we used in the utils.BytesBuffer. But I will set it :)